x-pack/test/functional/apps/uptime/overview.ts

```diff
@@ -65,10 +65,8 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         '0018-up',
         '0019-up',
       ]);
-      await retry.tryForTime(12000, async () => {
-        // there should now be pagination data in the URL
-        await pageObjects.uptime.pageUrlContains('pagination');
-      });
```

Comment:
Why is the retry now unnecessary?

Reply:
`pageUrlContains` already returns a `Promise`, and that `Promise` is a `retry` wrapper for this same functionality. See https://github.com/elastic/kibana/pull/60573/files/7a64af8ef560877440d0f77353809f5b16fd5824#diff-80a3c621caf51c25d10952feca0d6c73R63.

Effectively what we were doing before was `await retry(retry(check_url()))`.

Now we're just `await retry(check_url())`.